### PR TITLE
chore: enable annotations sourcemaps while linked

### PR DESCRIPTION
**Pull request by ConradJChan**
`yarn start:dev:linked` to run with annotations linked

### CODE REVIEW

File: build/webpack.common.config.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ const { BannerPlugin, DefinePlugin, NormalModuleReplacementPlugin } = require('w
 const license = require('./license');
 const pkg = require('../package.json');
 
+const isLinked = process.env.IS_LINKED === '1';
+const getAnnotationsPath = () =>
```
Comment (jstoffan) [inline build/webpack.common.config.js]: You can use `resolve.symlinks: true`, instead, I think. https://webpack.js.org/configuration/resolve/#resolvesymlinks

File: package.json [hunk 2]
```diff
@@ -141,6 +141,7 @@
         "setup": "yarn install",
         "start": "yarn setup && yarn build:dev --watch",
         "start:dev": "yarn build:i18n && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev yarn webpack-dev-server --config build/webpack.config.js",
+        "start:dev:linked": "f() { yarn link ${@:-box-annotations} && IS_LINKED=1 yarn start:dev; }; f",
```
Comment (jstoffan) [inline package.json]: Should this be `yarn start:linked`?

Comment (ConradJChan) [reply package.json]: Maybe I can add another for `yarn start:linked`

File: package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
         "babel-plugin-istanbul": "^5.2.0",
         "babel-plugin-react-remove-properties": "^0.3.0",
         "babel-plugin-transform-require-ignore": "^0.1.1",
-        "box-annotations": "^2.3.0",
+        "box-annotations": "^3.13.0",
```
Comment (jstoffan) [inline package.json]: We'll need to leave this as `^2.3.0` for now to avoid impacting customers.

**Pull request merged**